README.md

```diff
@@ -22,7 +22,7 @@ PerfCompare is hosted on heroku, and is updated every time commits are pushed to
 ## Setup
 
 ### Requirements
-
```

Comment:
Fixed act warnings appearing in the accessibility tests, 
Fixed one test was missing the view prop
also added a test for compare view in dark mode.